src/gui/parametersdialog.cpp

```diff
@@ -782,63 +863,27 @@ QString ParametersDialog::getErrorMessage(ErrorInfo &errorInfo) {
             break;
         }
         case ErrorLevelSyncPal: {
-            const auto &syncInfoMapIt = _gui->syncInfoMap().find(errorInfo.syncDbId());
-            if (syncInfoMapIt != _gui->syncInfoMap().end()) {
+            if (const auto &syncInfoMapIt = _gui->syncInfoMap().find(errorInfo.syncDbId());
```

Comment:
This feels less readable to me, is there any benefit to this change ?

Reply:
I understand your point, that's quite fair. There is indeed a loss in readability. I have followed the suggestion of Sonar Source here: The benefit is that the local variable is only used within the `if` clause where it belongs, with no risk of polluting / clashing / being wrong re-used of changed by external operations.

My view on this: I value more the locality gain than the readability loss (in this precise situation only, of course!).